pom.xml

```diff
@@ -35,12 +35,13 @@
   </scm>
 
   <properties>
-    <revision>6.20</revision>
+    <revision>7.0</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jetty.version>10.0.20</jetty.version>
+    <jetty.version>12.0.9</jetty.version>
     <slf4j.version>2.0.13</slf4j.version>
     <gitHubRepo>jenkinsci/winstone</gitHubRepo>
-    <spotless.check.skip>false</spotless.check.skip>
+    <maven.compiler.release>17</maven.compiler.release>
+    <maven.compiler.testRelease>17</maven.compiler.testRelease>
```

Comment:
As stated previously in https://github.com/jenkinsci/winstone/pull/341#discussion_r1593094003, unnecessary as of https://github.com/jenkinsci/pom/pull/491:

```suggestion
```